contracts/SCBridgeWallet.sol

```diff
@@ -125,7 +121,37 @@ contract SCBridgeWallet is IAccount {
     challengeExpiry = largestTimeLock + CHALLENGE_WAIT;
   }
 
-  function execute(address dest, uint256 value, bytes calldata func) external {
+  function challenge(
+    State calldata state,
+    bytes calldata ownerSignature,
+    bytes calldata intermediarySignature
+  ) external {
+    checkSignatures(state, ownerSignature, intermediarySignature);
+    _challenge(state);
+  }
+
+  function crossChainExecute(
+    uint executionChain,
+    uint[] calldata paymentChains,
+    address executeEntrypoint,
+    address[] calldata paymentEntrypoint,
```

Comment:
Some of this information (like entrypoints) in included in the calldata of this function so that the `validateUserOp` can access it when validating the UserOp